pkgs/by-name/tk/tkdiff/package.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl, tk, unzip }:
+
+stdenv.mkDerivation rec {
+  pname = "tkdiff";
+  version = "5.6";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${builtins.replaceStrings ["."] ["-"] version}.zip";
```

```suggestion
    url = "mirror://sourceforge/tkdiff/tkdiff-${builtins.replaceStrings ["."] ["-"] version}.zip";
```
it is nice to not replace pname wherever possible